scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/RestPageParameters.java

```diff
@@ -17,10 +18,16 @@
     private int offset;
     private int limit;
     private String tag;
+    private String sortAttribute;
+    private boolean ascendingOrder;
+
+
+
+    private SortSpecifierContainer sortParams;
 
     public RestPageParameters(SchedulerFrontendState frontEnd, String calledMethod, long from, long to,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Method has 12 parameters, which is greater than 7 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00107)